### PR TITLE
Update install-kafka-operator.md

### PR DESCRIPTION
**Pull request by baluchicken**
Update cert-manager and kubernetes version in the docs.

### CODE REVIEW

File: docs/install-kafka-operator.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ weight: 10
 
 {{< contents >}}
 
-The operator installs the 2.4.0 version of Apache Kafka, and can run on Minikube v0.33.1+ and Kubernetes 1.12.0+.
+The operator installs the 2.4.0 version of Apache Kafka, and can run on Minikube v0.33.1+ and Kubernetes 1.15.0+.
```
Comment (baluchicken) [inline docs/install-kafka-operator.md:9]:
```suggestion
The operator installs the 2.5.0 version of Apache Kafka, and can run on Minikube v0.33.1+ and Kubernetes 1.15.0+.
```

File: docs/install-kafka-operator.md [hunk 2]
```diff
@@ -47,14 +47,14 @@ Install cert-manager and CustomResourceDefinitions using one of the following me
...
-    kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml
+    kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.yaml
     ```
 
 - Using Helm:
 
     ```bash
     # Install CustomResourceDefinitions first
-    kubectl apply --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.11/deploy/manifests/00-crds.yaml
+    kubectl apply --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.15.1/deploy/manifests/00-crds.yaml
```
Comment (hi-im-aren) [inline docs/install-kafka-operator.md:57]: This link is 404

Comment (baluchicken) [inline docs/install-kafka-operator.md:57]:
```suggestion
```

File: docs/install-kafka-operator.md [hunk 3]
```diff
@@ -47,14 +47,13 @@ Install cert-manager and CustomResourceDefinitions using one of the following me
...
     ```bash
     # Install the CustomResourceDefinitions and cert-manager itself
-    kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml
+    kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.yaml
     ```
 
 - Using Helm:
 
     ```bash
     # Install CustomResourceDefinitions first
```
Comment (baluchicken) [inline docs/install-kafka-operator.md:56]:
```suggestion
```

File: docs/install-kafka-operator.md [hunk 4]
```diff
@@ -63,9 +61,9 @@ Install cert-manager and CustomResourceDefinitions using one of the following me
     # Install cert-manager into the cluster
     # Using helm3
     # You have to create the namespace before executing following command
-    helm install cert-manager --namespace cert-manager --version v0.11.0 jetstack/cert-manager
+    helm install cert-manager --namespace cert-manager --version v0.15.1 jetstack/cert-manager
```
Comment (baluchicken) [inline docs/install-kafka-operator.md:64]:
```suggestion
    helm install cert-manager --namespace cert-manager --create-namespace --version v0.15.1 jetstack/cert-manager
```

**Pull request merged**